<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSom</class>
 <widget class="QWidget" name="QSom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayoutMain">
    <item>
     <layout class="QVBoxLayout" name="menuVLayout" stretch="0,0">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Parameters</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="alfaLabel">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>alfa</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="alfaLEdit">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="gammaZeroLabel">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>gamma 0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="gammaZeroLEdit">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="sigmaZeroLabel">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>sigma 0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="sigmaZeroLEdit">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="networkSizeLabel">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>network size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="networkSizeLEdit">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>SOM initial data source</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Random Generation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_2">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>File</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>go n iterations forward</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="oneStepButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tenStepsButton">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>10</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="oneHundredStepsButton">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>100</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="twoHundredStepsButton">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>200</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="fiveHundredStepsButton">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>500</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
        <zorder></zorder>
        <zorder></zorder>
        <zorder></zorder>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="Simulation">
        <property name="title">
         <string>Simulation</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0,0,0,0">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <layout class="QHBoxLayout" name="simSpeedHLayout_2">
           <item>
            <widget class="QLabel" name="somMapScaleLabel">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>SOM map scale</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="somMapScaleHSlide">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>22</height>
              </size>
             </property>
             <property name="minimum">
              <number>50</number>
             </property>
             <property name="maximum">
              <number>300</number>
             </property>
             <property name="value">
              <number>150</number>
             </property>
             <property name="sliderPosition">
              <number>150</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="simSpeedHLayout">
           <item>
            <widget class="QLabel" name="simSpeedLabel">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>Simulation Speed</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="simSpeedHSlider">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>22</height>
              </size>
             </property>
             <property name="minimum">
              <number>400</number>
             </property>
             <property name="maximum">
              <number>5000</number>
             </property>
             <property name="value">
              <number>530</number>
             </property>
             <property name="sliderPosition">
              <number>530</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="invertedAppearance">
              <bool>false</bool>
             </property>
             <property name="invertedControls">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="iterationNumHLayout">
           <item>
            <widget class="QLabel" name="iterationNumberLabel">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>iteration number</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="iterationNumberLEdit">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>22</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="sigmaValueHLayout">
           <item>
            <widget class="QLabel" name="sigmaLabel">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>sigma</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="sigmaLEdit">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="gammaValHLayout">
           <item>
            <widget class="QLabel" name="gammaLabel">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>gamma</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="gammaLEdit">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pauseButton">
           <property name="text">
            <string>pause/resume</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="text">
            <string>START</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
